platform/machine/qemu/cluster.go

```diff
@@ -67,12 +67,16 @@ type MachineOptions struct {
 }
 
 type Disk struct {
-	Size   string // disk image size in bytes, optional suffixes "K", "M", "G", "T" allowed
-	Serial string // serial number to be passed to qemu via `serial=`. Disks show up under /dev/disk/by-id/virtio-<serial>
+	Size        string   // disk image size in bytes, optional suffixes "K", "M", "G", "T" allowed. Incompatible with BackingFile
+	BackingFile string   // raw disk image to use. Incompatible with Size.
+	DeviceOpts  []string // extra options to pass to qemu. "serial=XXXX" makes disks show up as /dev/disk/by-id/virtio-<serial>
 }
 
 var (
-	plog = capnslog.NewPackageLogger("github.com/coreos/mantle", "kola/platform/machine/qemu")
+	plog               = capnslog.NewPackageLogger("github.com/coreos/mantle", "kola/platform/machine/qemu")
+	ErrNeedSizeOrFile  = errors.New("Disks need either size or a backing file specified")
```

Comment:
These errors mix how they capitalize `size` & `backing file`, should probably just choose one case.